controlplane/internal/controllers/scale.go

```diff
@@ -60,8 +61,7 @@ func (r *RKE2ControlPlaneReconciler) initializeControlPlane(ctx context.Context,
 	}
 
 	bootstrapSpec := controlPlane.InitialControlPlaneConfig()
-	fd := controlPlane.NextFailureDomainForScaleUp()
-	if err := r.cloneConfigsAndGenerateMachine(ctx, cluster, rcp, bootstrapSpec, fd); err != nil {
+	if err := r.cloneConfigsAndGenerateMachine(ctx, cluster, rcp, bootstrapSpec); err != nil {
```

Comment:
We would probably want the failure domain logic still.

Reply:
reverted!